src/index.ts

```diff
@@ -33,39 +35,38 @@ export interface CrawlResult {
 }
 
 interface CrawlOptions {
-  url: string;
+  url: url;
   crawl: boolean;
   results: LinkResult[];
   cache: Set<string>;
   checkOptions: CheckOptions;
 }
 
-/**
- * Instance class used to perform a crawl job.
- */
+const isHtml = (response: gaxios.GaxiosResponse) => {
+  const contentType = response.headers['content-type'] || '';
+  return !!contentType.match('text/html');
+};
+
 export class LinkChecker extends EventEmitter {
-  /**
```

Comment:
Ah! Let's keep the jsdoc headers, they weren't hurting anything

Reply:
I've just learned never to trust jsdoc comments, unless they are statically checked for correctness. Otherwise they tend to become incorrect over time. Are these statically checked? I don't for a second disbelieve that your doc comments stay up to date, but most people (myself in particular) I would not trust that unless there's a static enforcement.